app/src/androidTest/java/com/sdp/movemeet/map/MapsActivityTest.java

```diff
@@ -80,6 +81,6 @@ public void mapsActivity_checkThatMarkersAreDisplayedAndClickable() throws UiObj
       / marker1.click();
         marker2.click();
         userMarker.click();
-        // Intents ? */
+        // Intents ?*/
```

Comment:
Comments can be clean if we don't need them

Reply:
That's right, it's a good practice!